samples/web/content/apprtc/js/appcontroller.js

```diff
@@ -58,6 +64,8 @@ var AppController = function(loadingParams) {
   this.remoteVideo_ = $(UI_CONSTANTS.remoteVideo);
   this.videosDiv_ = $(UI_CONSTANTS.videosDiv);
   this.roomLinkHref_ = $(UI_CONSTANTS.roomLinkHref);
+  
+  this.inCall_ = false;
```

Comment:
Not strictly necessary, but we will have to do something like this as we take the next step of having the app be able to make multiple calls without reloading. I can remove it for now if we want.